src/components/teasers/grid/teaser-grid.tsx

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
-import { up } from '../../breakpoint/breakpoint';
+import { up } from '../../style-utils/breakpoint';
+import { isNoneTouch } from '../../style-utils/is-touch';
 
 const ScrollContainer = styled.div`
-  overflow-x: auto;
+  overflow: hidden;
+
+  ${isNoneTouch()} {
```

Comment:
I love this 🥳 I think we should move the teaser grid in the teaser component. It's always used together so why split them up ? This is a great feature addition. Thank you!

Reply:
Thanks :)

Good idea to bring them closer together, will do.

Reply:
thanks!